power_grid_model_io_native_c/power_grid_model_io_native/include/power_grid_model_io_native/vnf_converter/vnf_pgm_converter.hpp

```diff
@@ -32,21 +34,22 @@ inline std::string serialize_data(power_grid_model::ConstDataset const& const_da
 
 class PgmVnfConverter {
   public:
-    PgmVnfConverter(char* buffer = nullptr, power_grid_model::WritableDataset* data = nullptr);
+    PgmVnfConverter(const char* buffer = nullptr, power_grid_model::WritableDataset* data = nullptr,
```

Comment:
don't use defaults if they actually are required
```suggestion
    PgmVnfConverter(const char* buffer, power_grid_model::WritableDataset* data,
```

if you really need it, use a separate default constructor `PgmVnfConverter() = default;`